### ISSUE TITLE
Choose proof-of-concept repository storage

### ISSUE THREAD

**Issue by dgkf** [pharmaR/thevalidatoR#6]
Although we've yet to settle on a technology for the production architecture, we've discussed having a minimalist proof-of-concept architecture that will let us move forward with the other modules.

For our pilot architecture, I think we should prioritize simplicity with the goal of allowing the pipeline to address challenges with managing a persistent repository and all the decisions that go along with that.

**Comment by dgkf** [comment]
A couple options that spring to mind:

### Storing a miniCRAN repo in a branch

One simple solution is to store a static package repository as part of the github repo. After a very quick (and certainly not exhaustive) search I found [this "static files storage" github action](https://github.com/marketplace/actions/static-files-storage) that could help us manage shared static storage across pipeline runs as a separate branch.

### Storing a repo on remote storage

If we prefer not to use a branch, I imagine we could do something similar with cloud storage, though this also introduces more administrative overhead.

### Posit Package Manager

Early in this project we chatted briefly with Posit who was open to the possibility of supporting this work with some testing infrastructure. The benefit of this is that PPM is probably more true-to-form of what an eventual solution could look like and offers tooling for interfacing via cli and api. The downside is that our tools would need to cater to this platform during early development and administering the platform would probably be more overhead than some of the other options.

--- 

Personally, I think the branch idea is looking like the most actionable path forward, but I'm curious to hear others' input

**Comment by yannfeat** [comment]
I agree, the static file storage is probably the best idea for our experimenting purposes.
